### DISCUSSION TITLE
MatrixCN tests

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/LMWG_dev#46] [General]
Place to discuss results from #45 to evaluate results from MatrixCN vs. AD spinup with a modern code base.

**Comment by wwieder** [comment]
### Plot of ecosystem, veg, and soil C pools from pSASU runs and the difference from pAD.
![image](https://github.com/NCAR/LMWG_dev/assets/8031012/41972030-6522-4c13-8839-5eda250b66a3)

**Comment by wwieder** [comment]
### Larger soil C pools in pAD appear to be from glut of passive C at depth in cold grids.

![image](https://github.com/NCAR/LMWG_dev/assets/8031012/4cf2e37d-09c8-4f6d-a070-c18398e15673)

**Comment by wwieder** [comment]
## Updated initializations
@olyson found a bug so that deep soil C was still being initialized in AD simulations.  
- updated plots with the bug fix are below, which seems to remove the deep soil C bulge in the pAD runs. 
- additional investigation for remaining differences likely need to be evaluated.
- NOTE, color bar on the difference map is an order of magnitude smaller than we had before.
![image](https://github.com/NCAR/LMWG_dev/assets/8031012/aa54edf7-2e9d-4f16-b799-3a0254234960)

![image](https://github.com/NCAR/LMWG_dev/assets/8031012/e5592e1c-fe4f-4d52-9e51-6b4adb9480a6)

**Reply by wwieder** [reply]
Depth profiles for a few additional high latitude grids:
![image](https://github.com/NCAR/LMWG_dev/assets/8031012/dc0a4d7d-6b51-4ee3-8a46-9582d557cc97)

**Reply by wwieder** [reply]
The persistence of these big bulges at depth on the pAD run (middle plots) makes me nervous.  maybe we can put our heads together and discuss?  I'm also at workshop almost all day weds-Thurs. I'll look and see if there's time on people's calendar's tomorrow sometime?

**Comment by olyson** [comment]
Standard diagnostics for the pSASU to pAD comparison are here:

https://webext.cgd.ucar.edu/I1850/ctsm51d159_f45_GSWP3_bgccrop_tsoiini_1850pSASU/lnd/ctsm51d159_f45_GSWP3_bgccrop_tsoiini_1850pSASU_51_60-ctsm51d159_f45_GSWP3_bgccrop_tsoiini_1850pAD_571_580/setsIndex.html

**Comment by olyson** [comment]
A couple of global sum vertical profile plots for soil carbon, the first showing the two soil temperature initialization simulations (274K and 272K) and the second showing the pSASU and 272K pAD simulations.

![soilc1d_global_ctsm](https://github.com/NCAR/LMWG_dev/assets/18671367/446101f6-70e2-4147-82f1-c988a0cd2863)

![soilc1d_global_pSASU_ctsm](https://github.com/NCAR/LMWG_dev/assets/18671367/49994742-14e5-41dd-a50e-22cfc43e7945)
